cmd/feed-ingress/main.go

```diff
@@ -31,23 +31,23 @@ var (
 
 func init() {
 	const (
-		defaultAPIServer    = "https://kubernetes:443"
-		defaultCaCertFile   = "/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-		defaultTokenFile    = "/run/secrets/kubernetes.io/serviceaccount/token"
-		defaultNginxConfDir = "."
-		defaultIngressPort  = 80
-		defaultNginxWorkers = 1
-		defaultHealthPort   = 12082
+		defaultAPIServer       = "https://kubernetes:443"
+		defaultCaCertFile      = "/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+		defaultTokenFile       = "/run/secrets/kubernetes.io/serviceaccount/token"
+		defaultNginxWorkingDir = "/"
```

Comment:
I was thinking defaults should match whatever it runs in the docker container as. So I thought it would be '/' in container. But actually I've put it in '/nginx' because it creates a ton of temp files. It doesn't really matter what the default is. I could swap it to '/nginx'. Were you using '.'?